cluster/gce/gci/configure-helper.sh

```diff
@@ -43,6 +43,39 @@ function setup-os-params {
   echo "core.%e.%p.%t" > /proc/sys/kernel/core_pattern
 }
 
+# secure_random generates a secure random string of bytes. This function accepts
+# a number of secure bytes desired and returns a base64 encoded string with at
+# least the requested entropy. Rather than directly reading from /dev/random, we
+# use uuidgen which calls getrandom(2) which verifies that the entropy pool has
+# been initialized sufficiently for the desired operation.
+#
+# ARGS:
+#   #1: number of secure bytes to generate. We round up to the nearest factor of 32.
+function secure_random {
+  local infobits="${1}"
+  if ((infobits <= 0)); then
+    echo "Requested insufficient entropy" 1>&2
```

Comment:
~~nit: everywhere else we just output errors & warnings to stdout~~
EDIT: realized that would conflict with the return value

Reply:
This might be a more informative message: `Invalid argument to secure_random: infobits=$infobits`